tensorflow/lite/micro/tools/ci_build/binary_size_test/binary_size_test.cc

```diff
@@ -0,0 +1,38 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "tensorflow/lite/micro/testing/micro_test.h"
+
+namespace {
+// Change this number to have a binary with a different
+// size of data section.
+constexpr int kSize = 64;
+long random_array[kSize] = {1, 2, 3, 4, 5, 6, 7, 8};
+}  // namespace
+
+TF_LITE_MICRO_TESTS_BEGIN
+
+TF_LITE_MICRO_TEST(BinarySizeChangeBykSize) {
+  // Just some code to create a binary and keep data section.
+  for (int i = 0; i < kSize; i++) {
+    random_array[i] = i + 1;
+  }
+
+  for (int i = 0; i < kSize; i++) {
+    TF_LITE_MICRO_EXPECT_EQ(random_array[i], i + 1);
```

Comment:
Won't this test fail for values of i > 7? They first 8 elements are initialized but kSize is 64.

Reply:
I think line #22 mislead: the only reason that I initialized the global array there is to put them in data section rather than bss section.  Because the uninitialized global variable goes to bss section and I suspect that due to that, changing kSize is probably not going to lead to binary size to change, which is the goal.

Reply:
so the test will not fail, b/c line 28 to line 30 re-populate the array up to kSize. 

Sorry for the confusion, I will add a comment in line #22.

Reply:
I will add comment in the code at the next PR. But I will add more comments in the PR description.

Reply:
Ahh got it, thanks.